### PR TITLE
adding liveness and readiness probes

### PR DESCRIPTION
**Pull request by deepak1725**
Adding Liveness and Readiness Probes
[CSSRE-2003](https://issues.redhat.com/browse/CSSRE-2003)
[CSSRE-2004](https://issues.redhat.com/browse/CSSRE-2004)

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -221,6 +221,20 @@ objects:
               name: web
               ports:
                 - containerPort: ${{GT_APP_PORT}}
+              readinessProbe:
+                exec:
+                  command:
+                    - cat
```
Comment (pepedocs) [inline openshift/template.yaml:227]: Does healthy mean "/tmp/healthy file exists" ? Otherwise what is the indication of healthy and unhealthy?

Comment (deepak1725) [reply openshift/template.yaml:227]: As per https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command

Comment (pepedocs) [reply openshift/template.yaml:227]: That looks like a live/ready/ness probe specific for busybox only.

**Review by pepedocs — CHANGES_REQUESTED** [review]
Can we please investigate what is the proper way to query for GT frontend health? If there is none then we need to create one or for staging we can probably just do an HTTP GET the root `/` url path.

**Comment by deepak1725** [comment]
> HTTP GET the root `/` url path

Makes sense. Will do the update

**Review by pepedocs — APPROVED** [review]
/lgtm

**Pull request merged**